scripts/xtensa-build-zephyr.py

```diff
@@ -76,25 +76,29 @@
 		"name": "apl",
 		"PLAT_CONFIG": "intel_adsp_cavs15",
 		"XTENSA_CORE": "X4H3I16w2D48w3a_2017_8",
-		"XTENSA_TOOLS_VERSION": f"RG-2017.8{xtensa_tools_version_postfix}"
+		"XTENSA_TOOLS_VERSION": f"RG-2017.8{xtensa_tools_version_postfix}",
+		"ZEPHYR_TOOLCHAIN_VARIANT": "xcc"
```

Comment:
```suggestion
		"DEFAULT_XCC_VARIANT": "xcc"
```

Reply:
Here I would remain with those names simply because they are "all wrong".
So basically there is no such thing as "xcc variant".
We have two drivers in Xtensa toolchains (well until RI-2021.7):
"xt-xcc" and "xt-clang".
Somebody in Zephyr project, called the "xt-xcc" driver as "xcc".
Then the Clang came out to the light and they had to distinguish between native POSIX LLVM Clang and Xtensa Clang.
So they named it "xcc-clang". Bad naming indeed but xt-clang is certainly not a variant of "xcc".
Closest to the true would be `DEFAULT_XTENSA_TOOLCHAIN_VARIANT` naming.
But again it would have values, which are not matching the actual driver names.
So I would just stick with ZEPHYR_TOOLCHAIN_VARIANT to avoid any further confusion.

Reply:
Very interesting, thanks for sharing! The main thing I was requesting is `DEFAULT` to show that it can be temporarily overriden with the environment, so people don't have uninstall/re-install a toolchain when using git checkout.

So how about `DEFAULT_TOOLCHAIN_VARIANT`? This script does not know anything besides `ZEPHYR`.